docs/network/net_binding_plugins/macvtap.md

```diff
@@ -121,40 +121,33 @@ exposed host interface (via the `lowerDevice` attribute, on the
 `macvtap-deviceplugin-config` `ConfigMap`).
 
 ## Macvtap network binding plugin
-[v1.1.1]
+
+> **Important**: The core macvtap binding was discontinued and removed in v1.3.0 (deprecated in v1.2.0). 
+> This documentation covers the **network binding plugin** implementation of macvtap.
 
 The binding plugin replaces the experimental core macvtap binding implementation
 (including its API).
```

Comment:
We can now remove this as it's covered in more specific detail in the previous line